src/data/courses.json

```diff
@@ -26,5 +26,9 @@
     {
         "title": "Laracasts",
         "link": "https://laracasts.com/"
+    },
+    {
+        "title": "LaraShout",
```

Comment:
I think it's better to add 'Larashout' to blogs, rather than in courses.

Reply:
Yeah, i move it to blogs from courses